CONTRIBUTING.md

```diff
@@ -24,6 +24,9 @@ PRs should follow the following guidelines
 - Should pass all system and unit tests run on GitHub Actions CI
 - Follow the code style guidelines bellow
 
+### Developer documentation
+
+We have some developer documentation (C++ classes
 
 ### Tests
 
```

Comment:
maybe worth removing the "Code Style" section below (but linking to it), so it's only in developer.md ?